src/components/Logo.js

```diff
@@ -4,7 +4,8 @@ const Logo = (props) => {
   return (
     <img
       alt="Logo"
-      src="/static/logo.svg"
+      src="/static/logowhite.png"
+      style={{width: '42px'}}
```

Comment:
example, here this has already been committed to master.

https://github.com/bundly/dash-beta/blob/9718d5682e94f5da5e5c3d5ee3cf66dcece6defa/src/components/Logo.js#L7-L9

Reply:
Closed as merged